### DISCUSSION TITLE
themes issue with tailwind

### DISCUSSION THREAD

**Discussion by atordvairn** [saadeghi/daisyui#568] [Q&A] [answered]
Hi, using Daisy ui in [this project](https://github.com/atordvairn/retrounhash)
and themes don't apply to some parts of the web page ( yes i have data-theme in `html`, and also don't use it explicitly on that element. ) 
I think it's tailwind interfering with the theme or something.

I am new to tailwind as well as Daisy ui

#### here's a screenshot with cyberpunk
![Screenshot_2022-03-09-22-38-35-485](https://user-images.githubusercontent.com/97218417/157496280-f1807c06-2cae-40be-a729-58b7c99f8d5a.jpeg)

**Comment by saadeghi** [comment, marked as answer]
Looks like there's some other CSS code applying colors. That might be the issue:

<img width="643" alt="Screen Shot 2022-03-09 at 8 31 24 PM" src="https://user-images.githubusercontent.com/7342023/157497880-80ed12d9-aa92-4f1f-9981-cb61898f710e.png">
